samples/wifi/radio_test/sample_description.rst

```diff
@@ -175,6 +184,392 @@ Testing
+
+              wifi_radio_test dpd 1
+
+
+         * To disable DPD hardware block, use the following command:
+
+              wifi_radio_test dpd 0
+
+         .. note::
+
+            * DPD command can be used only during tone transmission. For TX packet transmission, control of DPD hardware
+            * is taken care by firmware.
+
+
+         * To read chip temperature in degree Celsius, execute the following sequence of commands:
+
+           .. code-block:: console
+
+              wifi_radio_test get_temperature
+
+           The sample shows the following output:
+
+           .. code-block:: console
+
+               wifi_nrf: Temperature reading success:
+               wifi_nrf: The temperature is = 21 degree Celsius
+
+         * To measure RF RSSI status, with the following configuration:
+
+           * Channel: 144
+
+           Execute the following sequence of commands:
+
+           .. code-block:: console
+
+              wifi_radio_test rx 0
+              wifi_radio_test tx 0
+              wifi_radio_test init 144
+              wifi_radio_test get_rf_rssi
+
+           The sample shows the following output:
+
+           .. code-block:: console
+
+              wifi_nrf: RF RSSI value is = 0
+
+         .. note::
+
+            * RF RSSI Status is a 3-bit indicator. Higher the value of RF RSSI, higher is the level of saturation (non-linearity) at the LNA output.
+
+
+         * To compute optimal XO value, with the following configuration:
+
+           * Channel: 64
+
+           Execute the following sequence of commands:
+
+           .. code-block:: console
+
+              wifi_radio_test rx 0
+              wifi_radio_test tx 0
+              wifi_radio_test init 64
+              wifi_radio_test compute_optimal_xo_val
+
+           The sample shows the following output:
+
+           .. code-block:: console
+
+              wifi_nrf: Best XO value is = 43
+
+         .. note::
+
+            * This test requires a DUT to be connected with VSG using cabled setup. VSG should be continuously generating 0.5MHz tone.
+            * For channel 64, set the VSG frequency to 5320.5MHz. Set signal power level between -40dBm to -50dBm.
+
+
+
+         * To capture ADC samples with fixed RX gain, with the following configuration:
+
+           * Channel: 144
+           * LNA gain: 2
+           * BB gain: 10
+           * Capture length: 64
+
+           Execute the following sequence of commands:
+
+           .. code-block:: console
+
+              wifi_radio_test rx 0
+              wifi_radio_test tx 0
+              wifi_radio_test init 144
+              wifi_radio_test rx_lna_gain 2
+              wifi_radio_test rx_bb_gain 10
+              wifi_radio_test rx_capture_length 64
+              wifi_radio_test rx_cap 0
+
+           The sample shows the following output:
+
+           .. code-block:: console
+
+              ************* RX capture data ***********
+              E84FE4
```

Comment:
These values are not of interest since they will vary from run to run. Can we use some form of <- snip -> notation to only show a few lines ?

Reply:
Done.